electron/main.js

```diff
@@ -73,7 +75,7 @@ app.on('ready', createWindow);
 // Quit when all windows are closed.
 app.on('window-all-closed', function () {
   // On OS X it is common for applications and their menu bar
-  // to stay active until the user quits explicitly with Cmd + Q
+  // to stay state until the user quits explicitly with Cmd + Q
```

Comment:
What does this mean?

Reply:
That's a prime example of refactoring thinking it's really good at its job. Let me know if you find more, I'll fix them